frontend/src/components/DatePicker.tsx

```diff
@@ -61,30 +70,97 @@ function DatePicker(props: Props) {
     setLocaleRegistered(true);
   }, []);
 
-  let formGroupClassName = "usa-form-group";
+  let formGroupClassName = "usa-form-group grid-row";
   if (props.error) {
     formGroupClassName += " usa-form-group--error";
   }
 
   const className = `usa-input${props.className ? " " + props.className : ""}`;
 
+  const checkDate = (date: Date) => {
+    setErrors("");
+
+    if (props.maxDate) {
+      if (!isBefore(date, props.maxDate) && !isEqual(date, props.maxDate)) {
+        setErrors(
+          t("DatePicker.DateBefore") +
+            " " +
+            format(props.maxDate, props.dateFormat)
```

Comment:
I like better to use interpolation in the translation, this will give better context for the translator

"Select a date before {{date}}"

This is not a blocker but please update

Reply:
Done